Intro-JS/homework/homework.js

```diff
@@ -27,112 +27,157 @@ const newModulo = 21 % 5 === null ;
 
 function returnString(str) {
   //simply return the string provided: str
+  return str
 }
 
 function add(x, y) {
   // x and y are numbers
   // add x and y together and return the value
   // code here
-}
+    const sum = x + y;
+      return sum;
 
 function subtract(x, y) {
   // subtract y from x and return the value
   // code here
+    const diff = x - y;
+      return diff;
 }
 
 function multiply(x, y) {
   // multiply x by y and return the value
   // code here
+    const multiply = x * y;
+      return multiply;
 }
 
 function divide(x, y) {
   // divide x by y and return the value
   // code here
+  const divide = x / y;
+      return divide;
 }
 
 function areEqual(x, y) {
   // return true if x and y are the same
   // otherwise return false
   // code here
+  if(x===y)return true;
+  return false;
 }
 
 function areSameLength(str1, str2) {
   // return true if the two strings have the same length
   // otherwise return false
   // code here
+  if (str1.length == str2.length) {
+    return true;
+  }
+  return false;
 }
 
 function lessThanNinety(num) {
   // return true if the function argument: num , is less than ninety
   // otherwise return false
   // code here
+  if (num < 90) {
+    return true;
+  }
+  return false;
 }
 
 function greaterThanFifty(num) {
   // return true if num is greater than fifty
   // otherwise return false
   // code here
+   if (num > 50) {
+    return true;
+  }
+  return false;
+}
 }
 
 function getRemainder(x, y) {
   // return the remainder from dividing x by y
   // code here
+  const rem= x%y
+  return rem
 }
 
 function isEven(num) {
   // return true if num is even
   // otherwise return false
   // code here
+  if ((num % 2)==0) {
+    return true
+  }
+  return false
 }
 
 function isOdd(num) {
   // return true if num is odd
   // otherwise return false
   // code here
+  if ((num % 2) != 0) {
+    return true
+  }
+  return false
 }
 
 function square(num) {
   // square num and return the new value
   // hint: NOT square root!
   // code here
+  return Math.pow(num, 2)
 }
 
 function cube(num) {
   // cube num and return the new value
   // code here
+  return Math.pow(num, 3)
 }
 
 function raiseToPower(num, exponent) {
   // raise num to whatever power is passed in as exponent
   // code here
+  return Math.pow(num, exponent)
 }
 
 function roundNumber(num) {
   // round num and return it
   // code here
+  return Math.round(num)
 }
 
 function roundUp(num) {
   // round num up and return it
   // code here
+  return Math.ceil(num)
 }
 
 function addExclamationPoint(str) {
   // add an exclamation point to the end of str and return the new string
   // 'hello world' -> 'hello world!'
   // code here
+  str += "!";
```

Comment:
hello world is the string that need the ! , please adjust the values as stated in the comments